package.json

```diff
@@ -896,6 +906,11 @@
           "type": "boolean",
           "default": true,
           "description": "Use default template for playground files."
+        },
+        "mdb.uniqueObjectIdPerCursor": {
```

Comment:
It may be a bit redundent, but we've been keeping a list of the extension settings in the README as well:
https://github.com/mongodb-js/vscode/blob/6357efaabaf57a287f042087cec0f9598d789e18/README.md?plain=1#L67
Can we add this there to keep it consistent?